dashboard/src/components/api/che-workspace.factory.ts

```diff
@@ -635,6 +637,10 @@ export class CheWorkspace {
     return this.workspaceSettings ? this.workspaceSettings['che.workspace.auto_snapshot'] === 'true' : true;
   }
 
+  public getMasterApi(): CheJsonRpcMasterApi {
```

Comment:
IMHO I was thinking it was returning something else when I saw getMasterApi() in the code
It's because in che workspace we're already connected to the "master" node so it is looking odd to me to get a "master api" while it's mostly to handle RPC calls.